core/src/main/java/fj/P2.java

```diff
@@ -342,6 +344,34 @@ public B _2() {
     return (a, b) -> f.f(P.p(a, b));
   }
 
+  /**
+   * Polyomorphic lens targeted on _1.
+   */
+  public static <A, B, C> PLens<P2<A, B>, P2<C, B>, A, C> _1pLens() {
+    return pLens(__1(), a -> p2 -> P.p(a, p2._2()));
+  }
+
```

Comment:
no real convention here, I'm happy to change the name. I originally wanted _1Lens to be named __1 but that method already exist.
_1pLens being the polymorphic version of _1Lens.
Or should we use __1 and __2 for the lenses? not sure what would be the name for polymorphic version then...